src/Microsoft.DotNet.Cli.Utils/CommandResolution/IProject.cs

```diff
@@ -16,6 +16,8 @@ internal interface IProject
 
         string RuntimeConfigJsonPath { get; }
 
+        string OutputPath { get; }
```

Comment:
What do you think about calling this `FullOutputPath`?  It feels odd that this is named the exact same as an MSBuild property, but it doesn't use that MSBuild property....